src/components/Alert/Alert.tsx

```diff
@@ -66,10 +70,9 @@ export const Alert = (props: AlertProps) => {
                     {onClose && (
                         <Button
                             view="flat"
+                            className={bAlert('close-btn')}
                             onClick={onClose}
-                            extraProps={{
```

Comment:
Why change this?

Reply:
on each render created new object. You think better to leave it as it is?

Reply:
yes, the i18n call must be inside the component. Otherwise dynamic language change will not work.